quickwit-cli/src/index.rs

```diff
@@ -18,30 +18,26 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use std::collections::{HashSet, VecDeque};
-use std::fs::File;
 use std::io::{stdout, Stdout, Write};
 use std::path::PathBuf;
-use std::sync::Arc;
 use std::time::{Duration, Instant};
 use std::{env, fmt, io};
 
 use anyhow::{bail, Context};
-use byte_unit::Byte;
+use chrono::Utc;
 use clap::ArgMatches;
 use colored::Colorize;
 use itertools::Itertools;
-use json_comments::StripComments;
 use quickwit_actors::{ActorExitStatus, ActorHandle, ObservationType, Universe};
 use quickwit_common::uri::normalize_uri;
+use quickwit_config::{IndexConfig, IndexerConfig, SourceConfig};
 use quickwit_core::{create_index, delete_index, garbage_collect_index, reset_index};
-use quickwit_index_config::{match_tag_field_name, DefaultIndexConfigBuilder, IndexConfig};
-use quickwit_indexing::actors::{
-    IndexerParams, IndexingPipelineParams, IndexingPipelineSupervisor,
-};
-use quickwit_indexing::models::{CommitPolicy, IndexingDirectory, IndexingStatistics};
-use quickwit_indexing::source::{FileSourceParams, SourceConfig};
+use quickwit_index_config::match_tag_field_name;
+use quickwit_indexing::actors::{IndexingPipeline, IndexingPipelineParams};
+use quickwit_indexing::models::IndexingStatistics;
+use quickwit_indexing::source::FileSourceParams;
 use quickwit_metastore::checkpoint::Checkpoint;
-use quickwit_metastore::{do_checks, IndexMetadata, MetastoreUriResolver, Split, SplitState};
```

Comment:
@fulmicoton, I've disabled the checks because they were really getting in my way. You'll have to re-enable them.

Reply:
Created #841 856